pages/vi/profiles/garret123h.md

```diff
@@ -0,0 +1,20 @@
+![](https://placehold.it/350x90/009955/fff?text=Hello!) 
+
+my name is Garret Tullio, I am a junior at the University of Washington 
+and just starting my undergrad degree in computer science. Most of knowledge is in Java and Python. I have experience in 
+HTML, CSS, and Javascript and am excited to learn new software development. my time zone is **_pacific time_** and I am running on Mac OS.
+
+![Image](https://www.tacoma.uw.edu/sites/default/files/sections/CenterforBusinessAnalytics/Library_UWT.jpg)
+
+# What I'm Interested in
+
+* ![#ff0000](https://placehold.it/12/ff0000?text=+) Longboarding
+* ![#9900c5](https://placehold.it/15/9900c5?text=+) Spending time with the family
+* ![#157500](https://placehold.it/20/157500?text=+)Learning new things in school
+
+
+You Can Follow me On [![LinkedIn](https://content.linkedin.com/content/dam/me/business/en-us/amp/brand-site/v2/bg/LI-Bug.svg.original.svg)](https://www.linkedin.com/feed/)
```

Comment:
you put the link of linkedin feed :) It is not the link that someone can follow you on